e2e/go.mod

```diff
@@ -0,0 +1,115 @@
+module github.com/bloxapp/ssv/e2e
+
+go 1.20
+
+require (
+	github.com/attestantio/go-eth2-client v0.16.3
+	github.com/bloxapp/ssv v1.2.1-0.20231204141016-a6afbe837a26
```

Comment:
should the e2e be updated as well?

Reply:
Since it doesn't do proposals yet then there's a chance that it doesn't need to, right? @y0sher

Reply:
gonna try running it, not sure since go-eth2-client has been updated not just in proposals.